integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils_test.py

```diff
@@ -100,6 +101,22 @@ def test_unaltered_state(self, backend_name):
     # Test independent state.
     self.assertEqual([1.], module.get_count())
 
+  def test_to_mlir_type(self):
+    self.assertEqual('i8', tf_utils.to_mlir_type(np.dtype('int8')))
+    self.assertEqual('i32', tf_utils.to_mlir_type(np.dtype('int32')))
+    self.assertEqual('f32', tf_utils.to_mlir_type(np.dtype('float32')))
+    self.assertEqual('f64', tf_utils.to_mlir_type(np.dtype('float64')))
+
+  def test_save_input_values(self):
+    inputs = [
+        np.array([1, 2], dtype=np.int32),
+    ]
+    self.assertEqual('2xi32=1 2', tf_utils.save_input_values(inputs))
+    inputs = [
+        np.array([1, 2], dtype=np.float32),
+    ]
```

Comment:
```suggestion
    inputs = [np.array([1, 2], dtype=np.float32)]
```

Remove trailing comma and put onto single line since it fits.